src/components/datetime.js

```diff
@@ -19,6 +19,23 @@ module.exports = function(app) {
             }
           });
 
+          if($scope.component.defaultValue.length === 0) $scope.component.defaultValue = new Date();
+          else {
+            var dateVal = new Date($scope.component.defaultValue);
+
+            if(isNaN(dateVal.getDate())) {
+              try {
+                var Moment = moment();
```

Comment:
We should probably include moment as an angular dependency as angularMoment
Then do var moment = angularMoment here so we don't instantiate it. By instantiating it we don't allow them to pass any arguments to moment().